R/class-color.R

```diff
@@ -0,0 +1,84 @@
+#' R6 class for a Workbook Color
+#'
+#' A color
+#'
+#' @export
+wbColor <- R6::R6Class(
```

Comment:
As discussed, an `S3` object may be easy to do.  Can tag the `class`es as `openxlsx2` and `wbColor` for class checking.